src/Compilers/CSharp/Portable/Symbols/ConstructedNamedTypeSymbol.cs

```diff
@@ -49,6 +49,12 @@ protected override NamedTypeSymbol WithTupleDataCore(TupleExtraData newData)
         {
             throw ExceptionUtilities.Unreachable();
         }
+
+        internal override string ExtensionGroupingName
+            => _underlyingType.ExtensionGroupingName;
```

Comment:
It feels like the concept of grouping and marker name should only be applicable to definitions. Maybe we could avoid implementation here by adjusting callers (symbol display and docID). What do you think?

Reply:
I think the implementation is simple enough to not introduce additional complexity on consumer's side.